packages/data-broker/src/services/kobo/KoBoTranslator.js

```diff
@@ -12,10 +12,13 @@ export class KoBoTranslator {
 
   async fetchEntityInfoFromKoBoAnswer(koboEntityCode) {
     const entity = await this.models.entity.findOne({ code: `LA_sch_${koboEntityCode}` });
+    if (!entity) {
+      return { orgUnit: koboEntityCode, orgUnitName: koboEntityCode };
+    }
```

Comment:
If the translation layer can't find an entity, it passes through the raw value from kobo. We could possibly error or skip responses at this point, but I felt like that decision isn't really the duty of the translation layer.

Reply:
Yeah I think that's fair.